specification/consumption/resource-manager/readme.md

```diff
@@ -26,18 +26,27 @@ These are the global settings for the Consumption API.
 
 ``` yaml
 openapi-type: arm
-tag: package-preview-2019-04
+tag: package-preview-2019-05
 ```
 
 
+### Tag: package-preview-2019-05
+
+These settings apply only when `--tag=package-preview-2019-05` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2019-05'
+input-file:
+  - Microsoft.Consumption/preview/2019-05-01-preview/consumption.json
+```
 ### Tag: package-preview-2019-04
 
 These settings apply only when `--tag=package-preview-2019-04` is specified on the command line.
 
-```yaml $(tag) == 'package-preview-2019-04'
+``` yaml $(tag) == 'package-preview-2019-04'
```

Comment:
Is there any particular reason to add space?

Reply:
I did not do any manual editing here. I just got the latest version and used openapi to generate a new version 2019-05-01-preview. But I can remove it anyways